conf/switches.conf.example

```diff
@@ -21,7 +22,7 @@ isolationRole = isolation
 macDetectionRole = macDetection
 voiceRole = voice
 inlineRole = inline
-VoIPEnabled = no
+VoIPEnabled = N
```

Comment:
Since merging can mean a lot of things.

You can read how it works here
http://search.cpan.org/~shlomif/Config-IniFiles-2.88/lib/Config/IniFiles.pm#IMPORT_/_DELTA_FEATURES